style/javascript.md

```diff
@@ -0,0 +1,11 @@
+# Javascript
+
+Including NodeJS and client side scripting.
+
+## [StandardJS](http://standardjs.com/)
```

Comment:
Could we go with both then? The statement makes it clearer to me that we have adopted StandardJS as our style guide for JavaScript. But we could then include the links to help people understand why and how to use it.

> We have adopted [StandardJS](http://standardjs.com/) for all JavaScript written either in NodeJS apps or in client side scripting.
>
> Services that contain JavaScript should ensure it meets the standard. For more details about **StandardJS** see
>
> - [The rules](http://standardjs.com/rules.html)
> - [Why use a standard](http://standardjs.com/index.html#faq)
> - [How to use StandardJS](http://standardjs.com/#text-editor-plugins)

What'd ya 💭 ?